cpp-ch/local-engine/proto/extensions.proto

```diff
@@ -0,0 +1 @@
+../../../gluten-core/src/main/resources/substrait/proto/extensions.proto
```

Comment:
remove ?

Reply:
it's a link file. I don't want to add a extension struct into `substrait`, and put it an independent file